refactor(aqueduct): Return `IFluidDataStoreChannel` from `PureDataObjectFactory.instantiateDataStore` instead of `FluidDataStoreRuntime` implementation class. (#19353)
microsoft/FluidFramework: .changeset/bloated-ghosts-expand.md

---
"@fluidframework/aqueduct": major
---

> Return `IFluidDataStoreChannel` interface from `PureDataObjectFactory.instantiateDataStore` instead of `FluidDataStoreRuntime` implementation class.

Reviewer: Nit: a bit more focused on consumers of aqueduct as the readers of the release notes.
```suggestion
The return type of `PureDataObjectFactory.instantiateDataStore` is changing from `FluidDataStoreRuntime` to `IFluidDataStoreChannel`.
```
